### PR TITLE
Fixes #18406: Missing base class of plugins breaks `reflections` with caffeine version 2.8.2

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/18406

### CODE REVIEW

File: webapp/sources/pom.xml [hunk 1]
```diff
@@ -382,6 +382,7 @@ limitations under the License.
     <config-version>1.4.0</config-version>
     <cafeine-version>2.8.6</cafeine-version>
     <jgrapht-version>1.4.0</jgrapht-version>
+    <refections-version>0.9.12</refections-version>
```
Comment (ncharles) [inline webapp/sources/pom.xml:385]: typo: reflections-versions

Comment (ncharles) [reply webapp/sources/pom.xml:385]: maybe no s here

File: webapp/sources/rudder/rudder-web/pom.xml [hunk 2]
```diff
@@ -176,7 +176,7 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
     <dependency>
       <groupId>org.reflections</groupId>
       <artifactId>reflections</artifactId>
-      <version>0.9.12</version>
+      <version>${reflections-versions}</version>
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-web/pom.xml:179]: maybe no s here for version

**Comment by fanf** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

**Review by ncharles — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**